package.json

```diff
@@ -9,7 +9,7 @@
     "serve": "yarn --cwd oneapp-server live",
     "lint": "yarn --cwd oneapp-server lint",
     "dist": "yarn --cwd oneapp-server pkg",
-    "deploy:cf": "yarn --cwd oneapp-server deploy:pre && cf push usds-nj-oneapp-staging-api --strategy rolling",
+    "deploy:cf": "yarn --cwd oneapp-server deploy:pre && cf push usds-nj-oneapp-staging-api",
```

Comment:
Definitely some weird issues with GitHub changelogs, since that was a change I pushed last week. Interesting it's showing up in this PR.